webapp/packages/core-events/src/NotificationService.ts

```diff
@@ -171,7 +171,7 @@ export class NotificationService {
     return this.notify(notification, ENotificationType.Error);
   }
 
-  logException(exception: Error | GQLError | undefined | null, title?: string, message?: string, silent?: boolean): void {
+  logException(exception: Error | SessionError | GQLError | undefined | null, title?: string, message?: string, silent?: boolean): void {
```

Comment:
SessionError is instance of Error we don't need to specify it here (especially if we don't have code specific to this class)